AppServer/google/appengine/api/appcontroller_client.py

```diff
@@ -329,6 +329,19 @@ def add_routing_for_appserver(self, app_id, appserver_ip, port):
       app_id, appserver_ip, port, self.secret)
 
 
+  def remove_routing_for_appserver(self, version_key, ip, port):
```

Comment:
Do we need this to be exposed to the dashboard? From the look of it, I think it should only be an internal call between services instead of a callable function?